draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -65,6 +66,7 @@
   private boolean topViewResize;
   private boolean enableClickToMaximize;
   private boolean enableClickToMinimize;
+  private boolean enableTouchListener;
```

Comment:
We need a better name for this attribute. Instead of ```enableTouchListener``` don't you prefer to use ```enableTouch``` and methods to ```enableTouch(boolean enableTouch)``` and ```isEnabled```?